src/CSSBrush.ts

```diff
@@ -1,38 +1,70 @@
-import { RGB, createColorsFromMap, rgbHex } from 'color-map'
+import {RGB, createColorsFromMap, rgbHex} from 'color-map'
 
-import { rainbow } from './colors'
-import { BrushOption, Brush } from './interfaces'
+import {rainbow} from './colors'
+import {BrushOption, Brush} from './interfaces'
 
 export class CSSBrush implements Brush {
-  private count = 0
-  private colors: RGB[]
-  private map: { [index: string]: RGB } = {}
-  private option: BrushOption
+  private count = 0;
+  private colors: RGB[];
+  private map: { [index: string]: RGB } = {};
+  private option: BrushOption;
+  private gammaMap = {
+    rc: 0.2126,
+    gc: 0.7152,
+    bc: 0.0722,
+    lowc: 1 / 12.92
+  };
 
   constructor(option: Partial<BrushOption> = {}) {
     this.option = {
       maxColor: option.maxColor || 20,
       coloringText: option.coloringText || false
-    }
+    };
     this.colors = createColorsFromMap(rainbow, option.maxColor || 20)
   }
 
   color(id: string, ...rest: any[]) {
-    // TODO style (italic, bold, underscore) rotation
-    const rgb = this.getRgb(id)
-    const background = rgbHex(rgb)
-    const border = rgbHex(rgb.map(x => Math.max(0, x - 32)))
-    const color = rgb.every(x => x < 220) ? '#ffffff' : '#000000'
-    let idStr = `%c ${id} `
+    // #TODO style (italic, bold, underscore) rotation
+    const rgb = this.getRgb(id);
+    const background = rgbHex(rgb);
+    const border = rgbHex(rgb.map(x => Math.max(0, x - 32)));
```

Comment:
Yeah, I also rely on the tool (why not? 😄 ).
I haven't use webStorm for a while so I don't know if there is a way to easily turn off its config because when working on open source projects, each project's coding style can be different and I rely on the config and linting on the project to do that work.

In that regards vscode do a reasonably good job at that.